src/sentry/models/auditlogentry.py

```diff
@@ -195,6 +195,7 @@ class AuditLogEntry(Model):
     class Meta:
         app_label = "sentry"
         db_table = "sentry_auditlogentry"
+        indexes = [models.Index(fields=["organization", "datetime"])]
```

Comment:
Haha sorry, since we made this change the migration state also needs to be updated (probably just regenerate the migration)